tests/tt_metal/tt_metal/test_kernels/misc/watcher_waypoints.cpp

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+#include "ckernel.h"
+#include "debug/status.h"
+
+// Wait cycles to create a ~1sec delay
+#define WAIT_CYCLES 1200000000
+// Flag address for syncing between cores (BRISC controls this)
+#define SYNC_ADDR 409600
```

Comment:
Again, contradicting myself.  We could allocate an L1 buffer then pass the address in from the host either w/ a define or an argument

Reply:
Fixed this in latest commit